rer/subsites/browser/subsite_folder_utils.py

```diff
@@ -89,11 +95,11 @@ def get_subsite_attributes(self):
         subsite = self.get_subsite_folder()
         if not subsite:
             return {}
-        css_class = getattr(subsite, "subsite_css_class", subsite.getId())
+        css_class = getattr(subsite, 'subsite_css_class', subsite.getId())
         return {
             'title': subsite.Title(),
             'css_class': css_class,
-            'pin': '{}/++resource++opr.theme/pins/pin-{}.png'.format(
+            'pin': '{0}/++resource++opr.theme/pins/pin-{1}.png'.format(
```

Comment:
I feel like this is a leftover item from the opr custom theme we made last year... isn't it? :)

Reply:
yes, it's already fixed in the other branch (new_features)